### ISSUE TITLE
311 Data Weekly Engineering Meeting Agenda

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/311-data#1720]
### Overview
Meeting agendas for the weekly 311 Data Engineering member meetings, useful links and other resources

### Agenda Quick Links
[2024-05-08
](https://github.com/hackforla/311-data/issues/1720#issuecomment-2101751496)

### Resources/Instructions

[Engineering Onboard/Offboard Issue](https://github.com/hackforla/311-data/issues/1578)

Template for weekly meeting's comment.
```
## YYYY-MM-DD - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

```

**Comment by ryanfchase** [comment]
## 2024-05-08 - 311 Data Weekly Meeting
_[Back to Top](#)_

### Topic requests/questions:

1. Weekly Engineering agenda
  - we can briefly list all open tickets here, list PRs here, list goals here

2. Dev team announcements person: 
  - designated member to announce PRs, open tickets, etc (weekly rotation)
  - assign this person at the weekly meeting

3. Our plan to get new members:
  - create an Open Role on Engineering CoP
  - to justify this, we should make sure we have available tickets, and good onboarding + documentation

4. Bug-hunt tickets
  - a live forever ticket that gets reassigned every week, challenge to break the code
  - person assigned can do a bug writeup
  - can be a good first issue
  - Can document proposed solutions to these bugs and prioritize them for later

5. Contact Form
  - Create a Github access token for 311 Data dev

6. New Ticket Goals
  - try and maintain a number of Good First Issues (e.g. 3-5)
  - The number of available tickets should roughly match number of active devs

### Action Items:

### Resources / Links:

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake

**Comment by ryanfchase** [comment]
## 2024-05-18 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer
- @ryanfchase

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Open Github Issues](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3A%22Role%3A+Frontend%22+no%3Aassignee+-label%3ADependency+-label%3Adraft+-label%3AEpic)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake

**Comment by ryanfchase** [comment]
## 2024-05-22 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Open Github Issues](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3A%22Role%3A+Frontend%22+no%3Aassignee+-label%3ADependency+-label%3Adraft+-label%3AEpic)

### Topic requests/questions:

### Action Items:
- review Engineering Documentation
  - CONTRIBUTING.md
  - README.md

### Resources / Links:

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake

**Comment by ryanfchase** [comment]
## 2024-06-12- 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer
- (replace with @ github handle)

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Open Github Issues](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3A%22Role%3A+Frontend%22+no%3Aassignee+-label%3ADependency+-label%3Adraft+-label%3AEpic)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake

**Comment by ryanfchase** [comment]
## 2024-06-19 - 311 Data Weekly Engineering Meeting

## YYYY-MM-DD - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer
- (replace with @ github handle)

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)

- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)

### Topic requests/questions:

- [ ] What to do about converting the dreaded `{' '}` to `&nbsp;`
  - [ ] see Johnny's comment: https://github.com/hackforla/311-data/pull/1761#issuecomment-2169065261

### Action Items:

### Resources / Links:

- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew

**Comment by ryanfchase** [comment]
Leftover Action Items for @ryanfchase 
- [ ] Review website team wiki and see if we should adopt / reference it. (note: it will change soon!)
- [ ] The header should still be clickable while the loading modal is active
- [ ] First documentation ticket for high level data flow, high level tech stack
  - [ ] consider Excalidraw
- [ ] modify Eng Onboarding to add explicit link to quickstart

**Comment by ryanfchase** [comment]
## 2024-06-26 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer
- (replace with @ github handle)

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)

- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew

**Comment by ryanfchase** [comment]
RC TODO:
- Add dependency to this: https://github.com/hackforla/311-data/issues/1334
- blocked by: https://github.com/hackforla/311-data/issues/1778

**Comment by ryanfchase** [comment]
Meeting Notes from 8/31/24

- users thought calendar would expand after clicking the date
  - allow date dropdown to appear when clicking on date
  - clicking on "start date" should open month that contains "start date"
  - clicking on "end date" should open month that contains "end date"
  - OR clicking on anywhere on the text area emulates behavior of clicking on Calendar box
- users want to be able to enter the date
  - this may require much more work from design+dev to cover all "error states"
  - might be simpler to move forward with clickable textbox, then consider typed-out-dates

**Comment by ryanfchase** [comment]
## 2024-09-07- 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)

- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew

**Comment by ryanfchase** [comment]
Borrowing from Product Planning [new tickets comment](https://github.com/hackforla/311-data/issues/1528#issuecomment-2332981793).

New tickets checklist for eng:
- [ ] Add NC name pill to Search and Filters modal on address selection
- [X] Clicking on date range should drop down calendar: https://github.com/hackforla/311-data/issues/1806
- [X] Fix SR detail loading spinner: https://github.com/hackforla/311-data/issues/1805

**Comment by ryanfchase** [comment]
## 2024-09-14 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_


### Topic requests/questions:

### Action Items:

### Resources / Links:

- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Ryan C
- [ ] Johnny
- [ ] Shannan
- [ ] Billy Phan
- [ ] Drake
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew

**Comment by ryanfchase** [comment]
### Notes
- Pair programming
  - rubber ducky discussion
  - co-working zoom and/or slack

**Comment by ryanfchase** [comment]
## YYYY-MM-DD - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Next Week's Announcement Volunteer
- (replace with @ github handle)

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)

- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
## 2024-09-28 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)

- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by traycn** [comment]
## 2024-10-05 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog+%28ready+to+be+assigned%29%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
For next agenda

Need to discuss Vite build issue from VRMS (see if there is any overlap with our project)
- https://github.com/hackforla/VRMS/issues/1767

**Comment by ryanfchase** [comment]
## 2024-10-12 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
## 2024-10-12 Extended Session

### Notes

In our meeting we covered...
- https://github.com/hackforla/311-data/issues/1716
  - these metrics are useful, but ultimately they tell us what we already know: the browser is running out of memory for 3 months of data and onward

### Action Items

- [ ] How much data gets loaded on page-load?
  - pretty sure we just go to hugging face and look at how much data is in the 2024 data set
  - But is this what's causing the application to crash?
    - Guess 1: No.
      - It takes a long time to load the data and render the SRs, but this does not inherently crash the site
      - site will crash as soon as you start zooming in and out
      - we observe that there are MANY api calls to mapbox that occur as you zoom in/out of the map.

**Comment by ryanfchase** [comment]
## 2024-10-19 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
### Notes from 10/19

-  talked about maybe limiting RequestLayer's props, we are doing a comparison of requests against prev.requests, and this is potentially a really really bad performance action

**Comment by ryanfchase** [comment]
### Performance Notes (Cont)

I decided to try and see exactly how we were pulling 2023 data (since this isn't pulled on page-load). I noticed that it was actually only capable of pulling data from hugging face in 16.8MB sized chunks. See picture. This suggests that if we are to split up the data into manageable chunks, that 16.8MB is sort of the high end of resource size.

<details><summary>2023 parquet being organized into chunks</summary>
<p>

![image](https://github.com/user-attachments/assets/a4340282-b0a9-40fc-b7e7-2569d9be2b6f)


</p>
</details>

**Comment by ryanfchase** [comment]
## 2024-11-09 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
## 2025-01-18 - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
For next breakout / eng meeting
- Q: Do we have a ticket for defining test-cases for the app?
- A: Plans proposed in this epic: https://github.com/hackforla/311-data/issues/1768
  - ticket template for unit and integration tests
  - wiki page for testing, which could stage as a comprehensive list of user flows we wish to test
  - re: consult with UIUX + UXR to define a list of actions that we often test for
    - I prefer we provide UXD and UXR with a handful of basic examples, and that will serve as the template for these teams to contribute to this list

**Comment by traycn** [comment]
## YYYY-MM-DD - 311 Data Weekly Engineering Meeting
_[Back to Top](#)_

### Quick Links
- [Open PRs](https://github.com/hackforla/311-data/pulls)
- [Ready For Dev Lead](https://github.com/hackforla/311-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev+lead%22+-label%3Adraft+label%3A%22Role%3A+Frontend%22)
- [Bug Hunt Live-Forever Ticket](https://github.com/hackforla/311-data/issues/1726)
- [Frontend Project Board](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22)
  - [Issues Ready To Be Worked On](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=label%3A%22Role%3A+Frontend%22+no%3Aassignee+status%3A%22Prioritized+Backlog%22)
  - [What's Being Worked On (Frontend)](https://github.com/orgs/hackforla/projects/63/views/1?filterQuery=has%3Aassignee+label%3A%22Role%3A+Frontend%22)
  - [Developer Self Test](https://docs.google.com/presentation/d/1RUQk4mOrv4G1fja_D3GMWMtCyCqsdj64fNPlrIDJhxg/edit#slide=id.g114a9db5138_0_0)
- [Github Pages Dev Site](https://hackforla.github.io/311-data/#/map)
- [Engineering Onboarding Issue](https://github.com/hackforla/311-data/issues/1578)
- [311 Roster](https://docs.google.com/spreadsheets/d/1OsLDl7Ciwj7WjtzbgDz38g4kpOMNlUhdjlgzQxtQTvM/edit#gid=0)

### Topic requests/questions:

### Action Items:

### Resources / Links:

### Attendees

- [ ] Tracy
- [ ] Ryan C
- [ ] Shannan
- [ ] Billy Phan
- [ ] Danielle
- [ ] Kelly
- [ ] Andrew
- [ ] Brandon
- [ ] Dorian
- [ ] Mrudula

**Comment by ryanfchase** [comment]
Closing this ticket, we are using the 2025 ticket!
- https://github.com/hackforla/311-data/issues/1909
